tests/system/cli.go

```diff
@@ -256,14 +256,36 @@ func (c WasmdCli) WasmInstantiate(codeID int, initMsg string, args ...string) st
 	return addr
 }
 
+// Stake  delegates amount to given validator
+func (c WasmdCli) Stake(validatorAddr, amount string, args ...string) {
```

Comment:
🤔 We should be careful to not blow up the CLI with too many features. If this is some common command then let's add it here to make the tests more readable otherwise it is better to have them in the test itself.